homeassistant/components/plex/__init__.py

```diff
@@ -145,9 +145,21 @@ def _setup_plex(hass, config):
 
     entry.add_update_listener(async_options_updated)
 
-    hass.data[PLEX_DOMAIN][REFRESH_LISTENERS][server_id] = async_track_time_interval(
-        hass, lambda now: plex_server.update_platforms(), timedelta(seconds=10)
+    def update_plex():
+        async_dispatcher_send(hass, PLEX_UPDATE_PLATFORMS_SIGNAL.format(server_id))
+
+    session = async_get_clientsession(hass)
+    websocket = PlexWebsocket(plex_server, update_plex, session)
```

Comment:
If we pass the plexapi plex server instead of our wrapper plex server we don't need to define `url` method on our wrapper plex server.